src/lib/libast/tm/tmdata.c

```diff
@@ -189,6 +189,9 @@ static short	sum[] = { 0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334, 36
 
 static Tm_leap_t	leap[] =
 {
```

Comment:
I've verified the whole list of leap seconds (including your additions) against this IETF RFC: https://ietf.org/rfc/rfc8536.html#appendix-B.1 – and it all looks good. Thanks!